omod/src/main/java/org/openmrs/module/pihmalawi/web/taglibs/EMastercardAccessTag.java

```diff
@@ -186,19 +188,17 @@ protected String getNewMasterCardConfiguration(Form f) {
 		flowsheetForms.put(EncounterTypes.NCD_OTHER_INITIAL.name(), Arrays.asList("ncd_other_quarterly_screening","ncd_other_annual_screening","ncd_other_hospitalizations","ncd_other_visit"));
 		flowsheetForms.put(EncounterTypes.MENTAL_HEALTH_INITIAL.name(), Arrays.asList("mental_health_visit"));
         flowsheetForms.put(EncounterTypes.CHRONIC_CARE_INITIAL.name(), Arrays.asList("ncd_visit"));
-		flowsheetForms.put(HivMetadata.EXPOSED_CHILD_INITIAL, Arrays.asList("eid_visit"));
+		flowsheetForms.put(HivMetadata.EXPOSED_CHILD_INITIAL, Arrays.asList("eid_visit", "eid_test_results"));
+		flowsheetForms.put(EncounterTypes.VIRAL_LOAD_SCREENING.name(), Arrays.asList("viral_load_test_results"));
+		flowsheetForms.put(EncounterTypes.ART_INITIAL.name(), Arrays.asList("art_annual_screening", "art_visit"));
 
+		// hack to append the byConcept to the few forms that we fetch "byConcept" instead of by encounter type
+		// TODO: move this into a more organized customization
 		String byConcept = "";
 
 		if (f.getName().equals("Viral Load Tests")) {
-			headerForms.put(EncounterTypes.VIRAL_LOAD_SCREENING.name(), "blank_header");
-			flowsheetForms.put(EncounterTypes.VIRAL_LOAD_SCREENING.name(), Arrays.asList("viral_load_test_results"));
```

Comment:
I moved these up above because I didn't see any reason this needed to be in an if block and found this confusing... am I missing something?